modules/persistent-storage-local-tolerations.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-tolerations_{context}"]
+= Using tolerations with Local Storage Operator Pods
+
+You can control which nodes the Local Storage Operator Pod runs on and prevent
+other workloads from using those nodes by using tolerations on the Pods.
+
+You apply tolerations to the Local Storage Operator Pod through the LocalVolume Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node is a `key:value pair` that
+instructs the node to repel all Pods that do not tolerate the taint. Using a specific `key:value` pair
+that is not on other Pods ensures that only the Local Storage Operator Pod can run on that node.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to the {product title} nodes.
```

Comment:
I think we'd either want to include a Prerequisite saying that a node has been tainted that the Local Storage Operator needs to run on, or modify this line to say that:

> Local disks are attached to {product-title} nodes with a taint, and these nodes are expected to provision local storage.

Reply:
Amended second bullet and added third bullet in Prereqs.